src/scripts/jefferson

```diff
@@ -275,6 +338,9 @@ def scan_fs(content, endianness, verbose=False):
     fs = {}
     fs[JFFS2_NODETYPE_INODE] = {}
     fs[JFFS2_NODETYPE_DIRENT] = {}
+    fs[JFFS2_NODETYPE_XATTR] = {}
+    fs[JFFS2_NODETYPE_XREF] = {}
+    fs[JFFS2_NODETYPE_SUMMARY] = []
```

Comment:
Not needed because those node types are not used to reconstruct the filesystem on disk.